modules/admin_manual/pages/configuration/files/file_versioning.adoc

```diff
@@ -130,17 +130,27 @@ Do not apply any expiration rules. Delete all versions after 30 days.
 'versions_retention_obligation' => '30, 30',
 ----
 
-== Show the Author of a Version
+== Restoring of a Version
 
-You can enable and disable to show the author of a version in the versions history via an entry in config.php. For details see: xref:configuration/server/config_sample_php_parameters.adoc#save-and-display-the-author-of-each-version-of-uploaded-and-edited-files[Save and display the author of each version of uploaded and edited files]. This is helpful to keep track of who changed a version. 
+In the WebUI in the details view, there is a restore button next to the versions. It can be used to restore the file to a past version. Restore does not delete the old version, instead a new current version is created. 
 
-After adding the author, the version screen changes in the following way:
+WARNING: Note that retention policy can be applied after the restore operation, which could result in the old version being removed because of the retention policy. This behaviour can be changed with the persistent major version feature discussed in the section below.
```

Comment:
```suggestion
WARNING: Note that retention policy can be applied after the restore operation, which could result in the old version being removed because of the retention policy. This behavior can be changed with the persistent major version feature discussed in the section below.
```